DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    for iMonth in np.arange(0,n_months):
</pre><p>could use <code>enumerate</code> here and below to automatically increment j, i </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>